presto-main/src/main/java/com/facebook/presto/server/remotetask/HttpRemoteTaskFactory.java

```diff
@@ -186,6 +192,16 @@ else if (binaryTransportEnabled) {
         this.errorScheduledExecutor = newSingleThreadScheduledExecutor(daemonThreadsNamed("remote-task-error-delay-%s"));
         this.taskUpdateRequestSize = new DecayCounter(ExponentialDecay.oneMinute());
         this.taskUpdateSizeTrackingEnabled = taskConfig.isTaskUpdateSizeTrackingEnabled();
+        this.eventLoopGroup = new SafeEventLoopGroup(config.getRemoteTaskMaxCallbackThreads(),
```

Comment:
Will this spawn threads irrespective of whether eventloop is enabled or not ?

Wondering if we should initialize this eventLoopGroup only if isEventLoopEnabled==true.